test/integration/ads_integration_test.cc

```diff
@@ -1666,6 +1666,75 @@ TEST_P(XdsTpAdsIntegrationTest, Basic) {
   makeSingleRequest();
 }
 
+// TODO(adisuissa): Add a test with static cluster and load assignment, but dynamic endpoints.
```

Comment:
Yeah, we'll also need a bunch of unit tests exploring the corner case behaviors around warming and sharing of LEDS subscriptions across EDS subs.